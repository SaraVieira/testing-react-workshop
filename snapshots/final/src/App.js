import React from "react";
import List from "./List";
import "tachyons";

export default () => (
  <div className="sans-serif">
    <List />
  </div>
);
