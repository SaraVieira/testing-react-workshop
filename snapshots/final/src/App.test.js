/**
 * @jest-environment node
 */
import React from "react";
import { shallow, render } from "enzyme";

import App from "./App";

describe("<App />", () => {
    it("shallow", () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});
