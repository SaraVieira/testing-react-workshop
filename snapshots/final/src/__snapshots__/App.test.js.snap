// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> shallow 1`] = `
<div
  className="sans-serif"
>
  <Component />
</div>
`;
